src/pygeosx/pygeosx.cpp

```diff
@@ -347,11 +360,15 @@ PyInit_pygeosx()
 {
   import_array();
 
-  LvArray::python::PyObjectRef<> module{ PyModule_Create( &pygeosxModuleFunctions ) };
-  if( module == nullptr )
-  {
-    return nullptr;
-  }
+  PyObject * module = PyModule_Create( &pygeosxModuleFunctions );
```

Comment:
Can you replace these with `PyObjectRef`, similar to the previous implementation?  And try and catch the errors from module creation? Otherwise you risk leaks.

Reply:
Done.